src/licensedcode/data/licenses/hpnd-export-us.LICENSE

```diff
@@ -0,0 +1,32 @@
+---
+key: hpnd-export-us
+short_name: HPND with US Government export control warning
+name: HPND with US Government export control warning
+category: Free Restricted
+owner: Massachusetts Institute of Technology
+spdx_license_key: HPND-export-US
+other_urls:
+    - https://www.kermitproject.org/ck90.html#source
+ignorable_copyrights:
+    - Copyright (c) 1990 by the Massachusetts Institute of Technology
+ignorable_holders:
+    - the Massachusetts Institute of Technology
+---
+
+Copyright (C) 1990 by the Massachusetts Institute of Technology
+
+Export of this software from the United States of America may
```

Comment:
We have a perfect detection on this one which says this is a combination of: `generic-export-compliance AND mit-old-style-no-advert`

```
{
      "identifier": "generic_export_compliance_and_mit_old_style_no_advert-e085fc11-7044-ecee-48a1-9e8fb8131362",
      "license_expression": "generic-export-compliance AND mit-old-style-no-advert",
      "count": 1,
      "detection_log": [
        "not-combined"
      ],
      "matches": [
        {
          "score": 100.0,
          "start_line": 468,
          "end_line": 468,
          "matched_length": 36,
          "match_coverage": 100.0,
          "matcher": "2-aho",
          "license_expression": "generic-export-compliance",
          "rule_identifier": "generic-export-compliance_4.RULE",
          "rule_url": "https://github.com/nexB/scancode-toolkit/tree/develop/src/licensedcode/data/rules/generic-export-compliance_4.RULE"
        },
        {
          "score": 100.0,
          "start_line": 470,
          "end_line": 470,
          "matched_length": 96,
          "match_coverage": 100.0,
          "matcher": "2-aho",
          "license_expression": "mit-old-style-no-advert",
          "rule_identifier": "mit-old-style-no-advert_1.RULE",
          "rule_url": "https://github.com/nexB/scancode-toolkit/tree/develop/src/licensedcode/data/rules/mit-old-style-no-advert_1.RULE"
        }
      ]
```

Reply:
`mit-with-modification-obligations.LICENSE` has the following extra text when compared to this: 
```
Furthermore if you modify this software you must label
your software as modified software and not distribute it in such a
fashion that it might be confused with the original M.I.T. software.
```